src/MovieDetail.js

```diff
@@ -1,42 +1,113 @@
-import React from 'react'
+import React, { Component } from 'react'
+import ReactPlayer from 'react-player/lazy'  
 import './MovieDetail.css'
 
-const MovieDetail = ({ details }) => {
+class MovieDetail extends Component {
```

Comment:
Thank you for fixing our rendering problem and moving our selected state into movie detail